spec/integration/graphql/news_article_spec.rb

```diff
@@ -0,0 +1,212 @@
+      )
+
+      @world_location = create(
+        :live_edition,
+        document_type: "world_location",
+      )
+
+      @parent_taxon = create(
+        :live_edition,
+        document_type: "taxon",
+      )
+
+      @child_taxon = create(
+        :live_edition,
+        document_type: "taxon",
+      )
+
+      create(:link_set,
+             content_id: @child_taxon.document.content_id,
+             links_hash: {
+               parent_taxons: [
+                 @parent_taxon.document.content_id,
+               ],
+             })
+
+      create(:link_set,
+             content_id: @edition.document.content_id,
+             links_hash: {
+               government: [
+                 @government.document.content_id,
+               ],
+               organisations: [
+                 @organisation.document.content_id,
+               ],
+               people: [
+                 @person.document.content_id,
+               ],
+               taxons: [
+                 @child_taxon.document.content_id,
+               ],
+               topical_events: [
+                 @topical_event.document.content_id,
+               ],
+               world_locations: [
+                 @world_location.document.content_id,
+               ],
+             })
+    end
+
+    it "exposes news article specific fields in addition to generic edition fields" do
+      post "/graphql", params: {
+        query: <<~QUERY,
+          {
+              edition(
+                basePath: "/government/news/announcement",
+                contentStore: "live",
+              ) {
+                ... on Edition {
+                }
+                ... on NewsArticle {
+                  links {
+                    availableTranslations {
+                      basePath
+                      locale
+                    }
+                    government {
+                      details {
+                        current
+                      }
+                      title
+                    }
+                    organisations {
+                      basePath
+                      title
+                    }
+                    people {
+                      basePath
+                      title
+                    }
+                    taxons {
+                      basePath
+                      contentId
+                      title
+                      links {
+                        parentTaxons {
+                          basePath
+                          contentId
+                          title
+                        }
+                      }
+                    }
+                    topicalEvents {
+                      basePath
+                      title
+                    }
+                    worldLocations {
+                      basePath
+                      title
+                    }
+                  }
+                  basePath
+                  title
+                  details
```

Comment:
**nitpick:** line order

When reviewing/outside of a code editor, I've lost a bit of context on the nesting here. Might be useful to move these simple fields to the top, before the complicated links field block